eventmon/monitord/impl.go

```diff
@@ -37,6 +43,8 @@ func newMonitor(keymasterServerHostname string, keymasterServerPortNum uint,
 	webLoginChannel := make(chan string, bufferLength)
 	x509RawCertChannel := make(chan []byte, bufferLength)
 	x509CertChannel := make(chan *x509.Certificate, bufferLength)
+	Teelog := teelogger.New(One, Two)
```

Comment:
I think it's cleaner to create the two loggers here and then create the teelogger.